host/src/lib/HelperFunctions.jsx

```diff
@@ -183,6 +193,31 @@ export const getShortAnswers = (shortAnswerResponses) => {
   return { answersArray: [], confidenceArray };
 };
 
```

Comment:
Because we will be using the phase 1 short answers responses as possible answers in phase 2, we need to build an answer object for that purpose